NEWS.md

```diff
@@ -1,86 +1,64 @@
-3.18.1
+3.19.1
 ======
 
 ## Features
 
- * HTTP batching (#2135)
- * HTTP multi threading (#2291)
- * HTTP framing support for batches (#2190)
- * Python source (#2308)
- * Template support for Python destination (#2196)
- * HDFS time-reap (#2257)
- * Riemann batching (#2098)
- * flush-timeout() for HTTP (#2251)
- * Base64 encoding template function (#2223)
- * url-decode() template function (#2278)
- * Allow IPv4 mapped addresses in IPv6 mode (#2312)
- * app-parser: set ${.app.name} to the application identified (#2290)
- * Value pairs improvements (#2287)
- * syslog-ng-ctl config to print pre-processed configuration (#2280)
- * Add support for whitespace in template functions (#2271)
- * verify the configuration is the same as on the disk (#2345)
+ * HTTP load balancer (#2347)
+ * Slack destination (#2451)
+ * Add Cisco Catalyst formatted triplets support to cisco-parser() (#2394)
+ * Add RFC5424 syslog support to the system() source (FreeBSD 12.0 support) (#2430)
+ * Add network `interface()` option to network sources (#2389)
+ * Add so-reuseport() to network drivers (#2379)
+ * Enable supporting HTTP redirects (#2136)
+ * Configurable text separator for xml() parser (#2459)
```

Comment:
We just discussed this IRL and PR #2459 should not be merged just before OSE release 3.19.

Reply:
@MrAnno I force-pushed out this line.